estate/models/estate_property.py

```diff
@@ -0,0 +1,79 @@
+from odoo import fields, models, api
+from odoo.tools import date_utils as du
+from datetime import date
```

Comment:
nitpick: we like to have 2 different blocks of imports for python builtins and odoo internals.
I tend not to see abbreviations like `as du`.
```suggestion
from datetime import date

from odoo import fields, models, api
from odoo.tools import date_utils
```